motoman_motopos_mh1655_support/urdf/motopos_mh1655_macro.xacro

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" ?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:macro name="motoman_motopos_mh1655" params="prefix use_pedestal:=false">
+
+    <xacro:include filename="$(find motoman_resources)/urdf/common_materials.xacro"/>
+
+    <!-- link list -->
+    <link name="${prefix}base_link">
+      <visual>
+        <geometry>
+          <mesh filename="package://motoman_motopos_mh1655_support/meshes/motopos_mh1655/visual/base_link.stl"/>
+        </geometry>
+        <xacro:material_yaskawa_blue/>
+      </visual>
+      <collision>
+        <geometry>
+          <mesh filename="package://motoman_motopos_mh1655_support/meshes/motopos_mh1655/collision/base_link.stl"/>
+        </geometry>
+      </collision>
+      <xacro:if value="${use_pedestal}">
+        <!-- Pedestal is 0.695m high -->
+        <visual>
+          <geometry>
+            <mesh filename="package://motoman_motopos_mh1655_support/meshes/motopos_mh1655/visual/pedestal.stl"/>
+          </geometry>
+          <xacro:material_yaskawa_blue/>
+        </visual>
+        <collision>
+          <geometry>
+            <mesh filename="package://motoman_motopos_mh1655_support/meshes/motopos_mh1655/collision/pedestal.stl"/>
+          </geometry>
+        </collision>
+      </xacro:if>
+    </link>
+    <link name="${prefix}link_1">
+      <visual>
+        <geometry>
+          <mesh filename="package://motoman_motopos_mh1655_support/meshes/motopos_mh1655/visual/link_1.stl"/>
+        </geometry>
+        <material name="">
+          <color rgba="0.75294 0.75294 0.75294 1"/>
+        </material>
+      </visual>
+      <collision>
+        <geometry>
+          <mesh filename="package://motoman_motopos_mh1655_support/meshes/motopos_mh1655/collision/link_1.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <!-- end of link list -->
+
+    <!-- joint list -->
+    <joint name="${prefix}joint_1" type="revolute">
+      <parent link="${prefix}base_link"/>
+      <child link="${prefix}link_1"/>
+      <origin xyz="0 0 0.305" rpy="0 0 0"/>
+      <axis xyz="1 0 0"/>
+      <limit lower="${-2*pi}" upper="${2*pi}" effort="1274" velocity="1.3962"/>
```

Comment:
@EricMarcil: I'm having trouble finding documentation about the MH1655, but what I can find (`168961-1CD`) suggests the value for `velocity` here is too high.

Section `3.2.1` gives a maximum *headstock speed* of `10.8 rpm`, which should be roughly equivalent to `1.13` rad/s. Not `1.396`.

Am I looking at the wrong document (very well possible)?